### PR TITLE
Adding readme for enigma local setup

### PR DESCRIPTION
**Pull request by sharad-sharma**

### CODE REVIEW

File: docs/“How-to” guides/User Guides/Setup Enigma Locally/Local Setup without Docker.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ You should have python3 installed on your system and checked out the enigma repo
...
+The default [config.json.sample](https://github.com/browserstack/enigma/blob/main/config.json.sample) should be sufficient to start.
 
 You can then add module-specific configuration for the modules you want integrated with Enigma.
 For detailed instructions on configuration, follow [this doc](/docs/Configuration%20Guide.md)
 
 2. Add folder `db` in root folder of the repository.
 3. Add a package `access_modules` inside `Access`.
 
-Copy this [file](../../Access/base_email_access/access_modules_init.py) as `__init__.py` of `access_modules`
+Copy this [file](../../../../Access/base_email_access/access_modules_init.py) as `__init__.py` of `access_modules`
```
Comment (vedharish) [inline docs/“How-to” guides/User Guides/Setup Enigma Locally/Local Setup without Docker.md:18]: We can add absolute paths instead

```
Copy this [file](/Access/base_email_access/access_modules_init.py) as `__init__.py` of `access_modules`
```
in other places as well

**Review by vedharish — CHANGES_REQUESTED** [review]
Instead of naming the docs folder `Setup Enigma Locally`, lets rename to `Local Developer Setup`

Change the header in the file `docs/“How-to” guides/User Guides/Setup Enigma Locally/Local Setup with Docker.md` to `With Docker`

File: docs/“How-to” guides/User Guides/Setup Enigma Locally/Local Setup without Docker.md (hunk 1 above)
Comment (sharad-sharma) [reply docs/“How-to” guides/User Guides/Setup Enigma Locally/Local Setup without Docker.md]: Done.

File: README.md [hunk 2]
```diff
@@ -13,7 +13,8 @@ Refer to [this doc](/docs/%E2%80%9CHow-to%E2%80%9D%20guides/Adding%20Modules.md)
 
 The following steps are for hosting Enigma locally from published docker container images.
 
-For development setup, follow this [doc](/docs/one-click-dev.md)
+For development setup, follow these docs [using docker](/docs/“How-to”%20guides/User%20Guides/Setup%20Enigma%20Locally/Local%20Setup%20with%20Docker.md),
```
[the remark this answers is not in the document]
Comment (sharad-sharma) [reply README.md]: Done.

File: docs/“How-to” guides/User Guides/Local Developer Setup/Local Setup without Docker.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+For detailed instructions on configuration, follow [this doc](/docs/Configuration%20Guide.md)
+
+2. Add folder `db` in root folder of the repository.
+3. Add a package `access_modules` inside `Access`.
+
+Copy this [file](/Access/base_email_access/access_modules_init.py) as `__init__.py` of `access_modules`
+
+4. Add access modules inside `access_modules` to integrate them with Enigma. (Optional and not needed to get started)
+
+Copy each access module folder that you want to support manually here ensuring directory hierarchy in the following manner:
```
Comment (vedharish) [inline docs/“How-to” guides/User Guides/Local Developer Setup/Local Setup without Docker.md:22]:
```suggestion
Copy each access module folder that you want to support, ensuring directory hierarchy in the following manner:
```

**Review by vedharish — APPROVED** [review]

**Pull request merged**